public/css/nexthome.css

```diff
@@ -1,396 +1,683 @@
 @import url("https://fonts.googleapis.com/css?family=Inconsolata:400,700|Source+Sans+Pro:400,600");
```

Comment:
All assets should be locally hosted, including fonts.
We want to protect out users from 3rd party telemetry to the extend that we can.
Also, main.css already loads Inconsolata.
But if we need additional weights they should be added accordingly.

Reply:
added to main.css or kept in nexthome.css but hosted locally?
as per previous conversations wrt keeping main.css untouched unless absolutely required, I'll check if the other font weights are needed/used and if so, host them locally and load them in nexthome.css